tardis/io/logger/tests/test_logging.py

```diff
@@ -7,7 +7,10 @@
 from tardis import run_tardis
 import pytest
 
-pytestmark = pytest.mark.skip(reason='logging testing slow and disabled for now')
+pytestmark = pytest.mark.skip(
+    reason="logging testing slow and disabled for now"
+)
+
```

Comment:
I do not think this should be part of the PR.

Reply:
Maybe try rebasing this PR? Your PR should only make changes to solve issue https://github.com/tardis-sn/tardis/issues/1863. As @wkerzendorf said, please try to remove changes that do not belong to this PR.